rpm/kubelet.spec

```diff
@@ -169,7 +169,7 @@ mv cni-plugins/bin/ %{buildroot}/opt/cni/
 %{_sysconfdir}/kubernetes/manifests/
 
 %if %{KUBE_SEMVER} >= %{semver 1 11 0}
-%{_sysconfdir}/sysconfig/kubelet
+%config(noreplace) %{_sysconfdir}/sysconfig/kubelet
```

Comment:
how does this interact with L154?
```
install -p -m 644 -T kubelet.env %{buildroot}%{_sysconfdir}/sysconfig/kubelet
```

Reply:
Install copies the actual file to the staging directory. This line marks it as an entry in the rpm with its associated metadata.

Reply:
do we need the same for DEBs?

Reply:
I don't know the deb equivalent but would be surprised if it didn't have something like this.

Reply:
I think this is not needed for debian, as [the file in question](https://github.com/kubernetes/release/blob/a4cf32fd0b19f24974c9f5a3186ba6a344b159ef/debian/xenial/kubelet/lib/default/kubelet) is [installed into `/etc/`](https://github.com/kubernetes/release/blob/a4cf32fd0b19f24974c9f5a3186ba6a344b159ef/debian/xenial/kubelet/debian/kubelet.install#L3) and therefore is [automatically marked as a config file](https://www.debian.org/doc/manuals/maint-guide/dother.en.html#conffiles).